src/main/java/com/rocana/lucene/codec/v1/RocanaLucene50PostingsFormat.java

```diff
@@ -445,9 +445,9 @@ public FieldsConsumer fieldsConsumer(SegmentWriteState state) throws IOException
 
     boolean success = false;
     try {
-      FieldsConsumer ret = new BlockTreeTermsWriter(state, 
+      FieldsConsumer ret = new BlockTreeTermsWriter(state,
                                                     postingsWriter,
-                                                    minTermBlockSize, 
+                                                    minTermBlockSize,
```

Comment:
I do not think we should, as I'd like to be able to diff from the Lucene version and see what changed.